bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/action/EnergiDataServiceActions.java

```diff
@@ -100,7 +101,7 @@ public class EnergiDataServiceActions implements ThingActions {
     }
 
     @RuleAction(label = "@text/action.calculate-price.label", description = "@text/action.calculate-price.description")
-    public @ActionOutput(name = "price", type = "java.math.BigDecimal") @Nullable BigDecimal calculatePrice(
+    public @ActionOutput(label = "Price", type = "java.math.BigDecimal") @Nullable BigDecimal calculatePrice(
```

Comment:
Test result with snapshot 4356:

![image](https://github.com/user-attachments/assets/e5bed772-3ee2-431d-ac48-daef17225016)

```
2024-10-31 20:52:11.395 [WARN ] [der.AnnotationActionModuleTypeHelper] - Method EnergiDataServiceActions::getPrices has a single output but does not use the default output name in the ActionOutput annotation. This should be fixed in the binding.
```

Reply:
Probably because openhab/openhab-core#4435 is not included yet?

Reply:
Yes I guess.